prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -196,6 +196,12 @@ class HL7Reader(private val actionLogger: ActionLogger) : Logging {
     }
 
     companion object {
+        // map of HL7 message profiles: maps name of profile to configuration directory path
+        val messageProfileMap: Map<MessageProfile, String> = emptyMap()
```

Comment:
The mapping of message profile to configuration path is defined here. This is intentionally left empty for now.